organisaatio-ui/src/api/organisaatio.test.ts

```diff
@@ -23,8 +31,8 @@ const postinumerotKoodisto: Partial<Koodisto> = {
     uri2Arvo: (uri) => (uri ? '00530' : ''),
 };
 
-const yesterday = new Date(Date.now() - 86400000).toISOString().split('T')[0];
-const today = new Date().toISOString().split('T')[0];
+const Uiyesterday = moment(new Date(Date.now() - 86400000)).format('D.M.yyyy') as LocalDate;
```

Comment:
`moment().subtract(1, "days")`?

Reply:
i can change all date refecences ill find to moment.

Reply:
These are test files so do you think it matters?